packages/trader/src/sass/app/_common/drawer/positions-drawer-card.scss

```diff
@@ -145,6 +145,16 @@
             padding: 0.8rem 0rem;
         }
     }
+    &__currency-indicator {
```

Comment:
this css should probably be moved to the `CurrencyBadge` component

Reply:
Yes.